### PR TITLE
BeautifulSoup option to html2txt filter

### PR DESCRIPTION
**Pull request by lechuckcaptain**
Added BeautifulSoup library support to html2txt filter

### CODE REVIEW

File: lib/urlwatch/html2txt.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def html2text(data, method='lynx'):
     Method may be one of:
      'lynx' (default) - Use "lynx -dump" for conversion
      'html2text'      - Use "html2text -nobs" for conversion
+     'bs4'            - Use beautiful soap library to prettify the HTML
```
Comment (thp) [inline lib/urlwatch/html2txt.py]: Typo. Change the description to: "Use BeautifulSoup to prettify the HTML"

**Comment by thp** [comment]
Once the typo is fixed, this can go in.

**Comment by lechuckcaptain** [comment]
Fixed the typo.

**Comment by lkraav** [comment]
Can this go in now? :)

**Comment by lechuckcaptain** [comment]
Yes from my point of view. Let's just wait for @thp approval.

**Pull request merged**